code/modules/admin/admin_verbs.dm

```diff
@@ -437,9 +438,10 @@ var/list/admin_verbs_hideable = list(
 				new_key = copytext(new_key, 1, 26)
 			holder.fakekey = new_key
 			createStealthKey()
-			mob.invisibility = INVISIBILITY_MAXIMUM + 1 //JUST IN CASE
-			mob.alpha = 0 //JUUUUST IN CASE
-			mob.name = " "
+			if(isobserver(mob))
+				mob.invisibility = INVISIBILITY_MAXIMUM + 1 //JUST IN CASE
+				mob.alpha = 0 //JUUUUST IN CASE
+				mob.name = " "
```

Comment:
Makes sense. the invisibility would take care of that anyway so this is just a failsafe